feature/main/src/main/java/com/ku_stacks/ku_ring/main/notice/compose/inner_screen/LibraryNoticeScreen.kt

```diff
@@ -0,0 +1,112 @@
+package com.ku_stacks.ku_ring.main.notice.compose.inner_screen
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.ExperimentalMaterialApi
+import androidx.compose.material.pullrefresh.PullRefreshIndicator
+import androidx.compose.material.pullrefresh.PullRefreshState
+import androidx.compose.material.pullrefresh.pullRefresh
+import androidx.compose.material.pullrefresh.rememberPullRefreshState
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.dp
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import androidx.paging.compose.LazyPagingItems
+import androidx.paging.compose.collectAsLazyPagingItems
+import com.ku_stacks.ku_ring.designsystem.components.LazyPagingNoticeItemColumn
+import com.ku_stacks.ku_ring.designsystem.kuringtheme.KuringTheme
+import com.ku_stacks.ku_ring.domain.Notice
+import com.ku_stacks.ku_ring.main.R
+import com.ku_stacks.ku_ring.main.notice.CategoryNoticeViewModel
+
+@OptIn(ExperimentalMaterialApi::class)
+@Composable
+internal fun LibraryNoticeScreen(
+    shortCategoryName: String,
+    onNoticeClick: (Notice) -> Unit,
+    onNavigateToLibrarySeat: () -> Unit,
+    modifier: Modifier = Modifier,
+    viewModel: CategoryNoticeViewModel = hiltViewModel(key = shortCategoryName),
+) {
+    LaunchedEffect(shortCategoryName) {
+        viewModel.getNotices(shortCategoryName)
+    }
+
+    val noticesFlow by viewModel.noticesFlow.collectAsStateWithLifecycle()
+    val notices = noticesFlow?.collectAsLazyPagingItems()
```

Comment:
```suggestion
    val notices = viewModel.noticesFlow.collectAsLazyPagingItems()
```

이렇게 한줄로 줄일 수 있지 않을까요?

Reply:
`viewModel.noticesFlow`이 StateFlow 내부에 Flow를 포함하는 형태태여서 collect 후에 `.collectAsLazyPagingItem()` 적용이 가능한 것 같아요!